generators/server/templates/pom.xml.ejs

```diff
@@ -127,11 +127,11 @@
         <maven-war-plugin.version>3.2.2</maven-war-plugin.version>
         <jib-maven-plugin.version>0.9.11</jib-maven-plugin.version>
         <%_ if (!skipClient) { _%>
-        <frontend-maven-plugin.version>1.6</frontend-maven-plugin.version>
+        <frontend-maven-plugin.version>1.7.5</frontend-maven-plugin.version>
         <%_ } _%>
         <jacoco-maven-plugin.version>0.8.3</jacoco-maven-plugin.version>
         <lifecycle-mapping.version>1.0.0</lifecycle-mapping.version>
-        <scala-maven-plugin.version>3.4.4</scala-maven-plugin.version>
+        <scala-maven-plugin.version>4.0.1</scala-maven-plugin.version>
```

Comment:
I am just curious, but for what do we have the scala plugin here? I thought the only scala part is gatling (which is not compiled or executed with maven anymore)?

Reply:
I think it is for the profile `cc` :

```
Profile for doing "continuous compilation" with the Scala Maven plugin.
                It allows automatic compilation of Java classes as soon as they are saved.
                To use it, run in 3 terminals:
                - './mvnw -Pcc scala:cc' for continuous compilation of your classes
                - './mvnw -Pcc -Pdev' for hot reload of Spring boot
                - 'npm start/yarn start' for hot reload of the HTML/JavaScript asset
                Everything should hot reload automatically!
```

Reply:
+1 Thanks I remember